docs/kcl_mod.md

```diff
@@ -0,0 +1,70 @@
+# kcl.mod
+
+## Background
+kpm needs a configuration file for KCL project, which is responsible for marking the root directory, determining dependency versions, configuring compilation options or others.
+
+## Capacity
+
+- name: the name of the current project.
```

Comment:
Lack of structured specification definition of fields, including type, value range, etc.